app/scripts/background.js

```diff
@@ -61,6 +65,21 @@ initialize().catch(log.error)
 // setup metamask mesh testing container
 setupMetamaskMeshMetrics()
 
+/*
+ * The data emitted from the MetaMaskController.store EventEmitter, also used to initialize the MetaMaskController.
+ * @typedef MetaMaskState
+ */
+
+/*
+ * @typedef VersionedData
+ * @property {MetaMaskState} data - The data emitted from MetaMask controller, or used to initialize it.
+ * @property {Number} version - The latest migration version that has been run.
+ */
+
+/*
+ * Initializes the MetaMask controller, and sets up all platform configuration.
+ * @returns {Promise} Setup complete.
```

Comment:
Does this Promise resolve to a value? If so, it may be good to type that as well. This comment applies throughout.

Reply:
If it doesn't result in a value, is there a way to indicate it was deliberate to not include a result value?

Reply:
You can do `@returns {Promise<void>}` if you want to get really explicit about it, but mosts parsers infer this by now if no type is provided.